safe/report/expressions/map_report.py

```diff
@@ -208,7 +239,7 @@ def bearing_to_nearest_place(feature, parent):
 @qgsfunction(
     args='auto', group=label_group, usesGeometry=False, referencedColumns=[],
     help_text=help_message.to_html(), helpText=help_message.to_html())
-def name_of_the_nearest_place(feature, parent):
```

Comment:
This comment is for all expressions in this file, is-it not always `place` for the "exposure_key" ? Or can't we make it not compulsory? (ie keep the previous behavior make the new exposure_key field optional?)

Reply:
Ivan said we might be able to use it for volcanic ash as well. CMIIW @ivanbusthomi

Reply:
ok.
But can't we use this expression outside of the multiexposure context? (keep the previous behavior?)

Reply:
yes, we can 😃  I will make some changes first

Reply:
ok cool. let me know if it's possible